src/bundle/Resources/views/content/locationview.html.twig

```diff
@@ -13,6 +13,10 @@
 
 {% block body_class %}ez-content-view ez-has-two-sidebars{% endblock %}
 
+{% block meta %}
+<meta name="CurrentLocationId" content="{{ location.id }}" />
```

Comment:
@webhdx are you ok with this approach of exposing meta information for React-based scripts?

Reply:
Wouldn't passing such information to props upon component creation be a better solution than this? I mean component themselves shouldn't access their context directly.

Reply:
I don't mind this